bears/coffee_script/CoffeeLintBear.py

```diff
@@ -23,7 +23,10 @@ def convert_if_not_empty(value: str, conversion):
 @linter(executable='coffeelint')
 class CoffeeLintBear:
     """
-    Coffeelint's your files!
+    Use ``coffeelint`` to parse CoffeeScript code to help keep the code
+    clean and consistent and raise style inconsistencies.
```

Comment:
how about

```
Check CoffeeScript code for a clean and consistent style.
```

shorter and it's the info the user actually wants IMO, keep the for more info paragraph